syft/workers/base.py

```diff
@@ -206,14 +206,7 @@ def recv_msg(self, bin_message):
         # Step 1: route message to appropriate function
         response = self._message_router[msg_type](contents)
 
-        # # Step 2: If response in none, set default
-        # TODO: not sure if someone needed this - if this comment
-        # is still here after Feb 15, 2018, please delete these
-        # two lines of (commented out) code.
-        # if response is None:
-        #     response = None
-
-        # Step 3: Serialize the message to simple python objects
```

Comment:
would be cool to still keep this "Step 3:" piece

Reply:
Thank you @iamtrask 
I marked it as # Step 2 since the previous # Step 2 was deleted. Should it still appear as # Step 3 then?

Reply:
So I kept it but as Step 2 not 3, is it ok?